docs/Integrations/maps/google-maps/generate-maps-keys.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Generate Maps Keys
+sidebar_position: 1
+
+---
+
+# Maps & Places APIs
+
+FlutterFlow natively supports **Google Maps**, providing a seamless and efficient way to embed
+interactive maps into your FlutterFlow apps. It also supports **Places API** that returns formatted location data and imagery about establishments, geographic locations, or prominent points of interest.
+
+## Add Maps APIs
+
+To enable **Google Maps** in your project, please follow the steps:
+
+### Get API Keys
+
+To start working with **Google Maps APIs**, you need to enable the **Maps API** from the [Google
+Cloud Console](https://console.cloud.google.com/).
+
+- As you land on the Cloud console, make sure you are in the correct Google Cloud project. Then,
```

Comment:
Would be better to have the steps as an ordered list.

Reply:
Fixed